src/redux/stores/config/configHelpers.ts

```diff
@@ -0,0 +1,35 @@
+import { TokenKinds } from '@airswap/constants';
+import erc20AbiContract from '@openzeppelin/contracts/build/contracts/ERC20.json';
+import erc721AbiContract from '@openzeppelin/contracts/build/contracts/ERC721.json';
+import { ethers } from 'ethers';
+
+export const getCollectionTokenTokenKind = async (provider: ethers.providers.Web3Provider, address: string): Promise<TokenKinds | undefined> => {
```

Comment:
can probably rename this from `getCollectionTokenTokenKind` to `getCollectionTokenKind`